tendrl/gluster_integration/sds_sync/georep_details.py

```diff
@@ -177,20 +178,21 @@ def aggregate_session_status():
             vol_id = volume.vol_id
             sessions = None
             try:
-                sessions = NS.tendrl.objects.GeoReplicationSession(
-                    vol_id=volume.vol_id
-                ).load_all()
+                sessions = etcd_utils.read(
```

Comment:
Why we are changing this? we can do a load on the object itself right?

Reply:
GeoReplicationSession object directry present inside /cluster/{cid}/Volumes/{vid}/GeoRepSessions also in same directry we have GeoReplicationPair (/cluster/{cid}/Volumes/{vid}/GeoRepSessions/{geo_rep_id}/GeoReplicationPair). So we are still not tested to use load all with directory which have child directory. 

if /cluster/{cid}/Volumes/{vid}/GeoRepSessions/{geo_rep_id} does not have any child directory then load_all with work without any problem. But directory with subdirectory still we need to test.

So I and @shtripat decided to use `ectd.utils.read`  to find geo_rep_session ids and using ids we are loading GeoReplicationPair because GeoReplicationPair does not have any subdirectory.

Reply:
@nthomas-redhat so by this time GeoReplicationSession objects are not yet written and we are writing here. To get the session_id we are doing a raw read from path written by GeoReplicationPair objects and then forming the session object down and writing.

Reply:
ok